pages/uitgifte-email.md

```diff
@@ -24,14 +24,20 @@ Wanneer u op die link klikt ontvangt u het e-mail attribuut.
 
 (Het blijkt dat in sommige e-mail lezers op mobiele telefoons deze
 link niet goed "klikbaar" weergegeven wordt. U kunt dan proberen de
-link zelf te copieëren in een webbrowser. U kunt ook proberen de mail
+link zelf te kopiëren in een webbrowser. U kunt ook proberen de mail
 op een ander apparaat, zoals een laptop of PC, te lezen. Er wordt naar
 gekeken hoe dit probleem het beste opgelost kan worden.)
 
 De geldigheid van dit e-mail attribuut is *een jaar*.
 
-De stichting Privacy by Design verwijdert dit e-mail adres na uitgifte
-uit de eigen systemen. De stichting houdt geen log bij van de
-uitgifte.
+**Note:**
+Vanaf 30 November 2020 worden e-mail attributen niet meer uitgegeven
+door de stichting Privacy by Design, maar door
+[SIDN](https://www.sidn.nl).
+
+SIDN verwijdert dit e-mailadres na uitgifte uit de systemen.
+In kader van onderhoud en monitoring houdt SIDN een log bij van de uitgifte.
+Deze log wordt automatisch verwijderd na een verloop van tijd.
```

Comment:
Volgens mij moet het "Dit log" zijn.

Reply:
Dit mag allebei, omdat het een Engels leenwoord is :) https://onzetaal.nl/taaladvies/engelse-woorden-de-het-weblog/#:~:text=Het%20is%20allebei%20juist.,%C3%A9n%20als%20een%20het%2Dwoord.

Persoonlijk vind ik 'deze' beter klinken, maar dat is een kwestie van wat je gewend bent denk ik.

Reply:
Oplossing: `Dit wordt automatisch verwijderd ...` 😉

Reply:
Ik had "het logbestand wordt automatisch verwijderd na verloop van tijd" (Weet niet of een verloop van tijd iets is)